lib/govuk_index/presenters/details_presenter.rb

```diff
@@ -39,6 +39,21 @@ def image_url
       details.dig("image", "url")
     end
 
+    def parts
+      parts = details.fetch("parts", [])
+      return unless parts && parts.any?
+
+      parts.map do |part|
+        body = part.fetch("body", [{}]).first
```

Comment:
I looked at [the universal credit content item](https://www.gov.uk/api/content/universal-credit), and its parts have the schema `{"title": "...", "slug": "...", "body": "..."}`, not `"body": [{"content": "..."}]`.

Reply:
Yeah that tripped me up, too. The way these get represented to search api (`body: [{ content ...`] is different to how they look in the content store.